### PR TITLE
Upload artifact from buildkite to Google storage and post the artifact media url to the pull request.

### PR DESCRIPTION
**Pull request by mrpau-eduard**
## Summary

Enable the artifact to be downloadable each time we have a pull request and commits.
Once the buildkite done building the `Python Packages` and the `Installers`. It will upload the artifacts in the Google Cloud and post a comment on the pull requester with the media `URL` pointing on the artifacts.

## TODO

- [x] Upload artifact to Google cloud storage via script
- [x] Post a comment Github PR for the media links
- [x] integrate  script to buildkite
- [x] Documentation.

## Screenshots (if appropriate)

screen shot 2017-06-04 at 4 58 07 am

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -12,5 +12,12 @@ steps:
   - label: Build the pex file
     command: mkdir -p dist && .buildkite/build_pex.sh
 
+  - wait
```
Comment (aronasorman) [inline .buildkite/pipeline.yml]: Do we need a wait on this? It would be nice if installer building can run in parallel as much as possible.

**Comment by aronasorman** [comment]
Woo!

**Comment by aronasorman** [comment]
Buildkite is failing though.

**Comment by mrpau-eduard** [comment]
## Build Artifacts
**Kolibri Installers**
Windows Installer: [KolibriSetup-VF-0.3.1b3.dev25.exe](None)

**Python packages**
Pex: [None](None)
Whl file: [kolibri-0.3.1b3-py2.py3-none-any.whl](None)
Zip file: [None](None)
Tar file: [None](None)

**Comment by mrpau-eduard** [comment]
## Build Artifacts
**Kolibri Installers**
Windows Installer: [KolibriSetup-VF-0.3.1b3.dev25.exe](None)

**Python packages**
Pex: [None](None)
Whl file: [kolibri-0.3.1b3-py2.py3-none-any.whl](None)
Zip file: [None](None)
Tar file: [None](None)

**Comment by mrpau-eduard** [comment]
## Build Artifacts
**Kolibri Installers**
Windows Installer: [KolibriSetup-VF-0.3.1b3.dev25.exe](None)

**Python packages**
Pex: [None](None)
Whl file: [kolibri-0.3.1b3-py2.py3-none-any.whl](None)
Zip file: [None](None)
Tar file: [None](None)

**Comment by mrpau-eduard** [comment]
## Build Artifacts
**Kolibri Installers**
Windows Installer: [KolibriSetup-VF-0.3.1b3.dev25.exe](https://www.googleapis.com/download/storage/v1/b/le-downloads/o/kolibri%2Fbuildkite%2Fbuild-1589%2FKolibriSetup-VF-0.3.1b3.dev25.exe?generation=1496799329244511&alt=media)

**Python packages**
Pex: [None](None)
Whl file: [kolibri-0.3.1b3-py2.py3-none-any.whl](https://www.googleapis.com/download/storage/v1/b/le-downloads/o/kolibri%2Fbuildkite%2Fbuild-1589%2Fkolibri-0.3.1b3-py2.py3-none-any.whl?generation=1496799297747538&alt=media)
Zip file: [None](None)
Tar file: [None](None)

**Comment by mrpau-eduard** [comment]
## Build Artifacts
**Kolibri Installers**
Windows Installer: [KolibriSetup-0.4.0b5.dev83.exe](https://www.googleapis.com/download/storage/v1/b/le-downloads/o/kolibri%2Fbuildkite%2Fbuild-1589%2FKolibriSetup-0.4.0b5.dev83.exe?generation=1496799764642544&alt=media)

**Python packages**
Pex: [kolibri-v0.4.0-beta5-83-g4f77162.pex](https://www.googleapis.com/download/storage/v1/b/le-downloads/o/kolibri%2Fbuildkite%2Fbuild-1589%2Fkolibri-v0.4.0-beta5-83-g4f77162.pex?generation=1496799750150554&alt=media)
Whl file: [kolibri-0.4.0b5.dev83-py2.py3-none-any.whl](https://www.googleapis.com/download/storage/v1/b/le-downloads/o/kolibri%2Fbuildkite%2Fbuild-1589%2Fkolibri-0.4.0b5.dev83-py2.py3-none-any.whl?generation=1496799744255880&alt=media)
Zip file: [kolibri-0.4.0b5.dev83.zip](https://www.googleapis.com/download/storage/v1/b/le-downloads/o/kolibri%2Fbuildkite%2Fbuild-1589%2Fkolibri-0.4.0b5.dev83.zip?generation=1496799738446693&alt=media)
Tar file: [kolibri-0.4.0b5.dev83.tar.gz](https://www.googleapis.com/download/storage/v1/b/le-downloads/o/kolibri%2Fbuildkite%2Fbuild-1589%2Fkolibri-0.4.0b5.dev83.tar.gz?generation=1496799732455115&alt=media)

**Comment by mrpau-eduard** [comment]
Hi @aronasorman can you take a look again on my `PR`. I am done updating the script.
The above [comment](https://github.com/learningequality/kolibri/pull/1589#issuecomment-306696873) is the example output of this `PR`. 

Can we create another Github account to use? Its my dummy account for Github.

**Comment by aronasorman** [comment]
Woo! Let's get this in now so everyone can get this.

**Pull request merged**